tutorials/platform/customizing_html5_shell.rst

```diff
@@ -133,6 +133,23 @@ them by calling the :js:meth:`engine.preloadFile` method with a path to a file o
 with an ``ArrayBuffer`` object. In case of the ``ArrayBuffer``, or one of its views, a second argument
 must be specified to define an internal path for the loaded resource.
 
+Here is a short example for loading additionals files. In this case the file "my_text.txt" is loaded in
+when while the engine starts up.
+::
+    Engine.load(MAIN_PACK)
+    engine.preloadFile('$GODOT_BASENAME.js')
+    engine.preloadFile('$GODOT_BASENAME.pck')
+    engine.preloadFile('$GODOT_BASENAME.png')
+    engine.preloadFile('$GODOT_BASENAME.wasm')
+    engine.preloadFile('my_text.txt')
```

Comment:
This is wrong, you only need to preload the `pck` and your file (not the PNG, not the wasm, not the js).

Alternatively , you should be able to skip the `Engine.load(EXECUTABLE_NAME)`, `engine.preloadFile(MAIN_PACK)`, and `engine.init()` by using startGame:

```javascript
engine.preloadFile('my_text.txt').then(function() {
    engine.startGame(EXECUTABLE_NAME, MAIN_PACK).then(() => {
        setStatusMode('hidden');
        initializing = false;
    }, displayFailureNotice);
});
```